src/trio/_tests/_check_type_completeness.json

```diff
@@ -20,13 +20,10 @@
     "all": [
         "No docstring found for class \"trio.MemoryReceiveChannel\"",
         "No docstring found for class \"trio._channel.MemoryReceiveChannel\"",
-        "No docstring found for function \"trio._channel.MemoryReceiveChannel.statistics\"",
-        "No docstring found for class \"trio._channel.MemoryChannelStats\"",
-        "No docstring found for function \"trio._channel.MemoryReceiveChannel.aclose\"",
+        "No docstring found for class \"trio.MemoryChannelStatistics\"",
+        "No docstring found for class \"trio._channel.MemoryChannelStatistics\"",
```

Comment:
I guess you could've added a docstring to the class at the same time to get rid of these as well